### ISSUE TITLE
Use storagestate for authentification?

### ISSUE THREAD

**Issue by LisBerndt** [abhinaba-ghosh/playwright-lighthouse#49]
Hello! 
Is it possible to simply use the storagestate and reuse my token-data of my user.json as described [here](https://playwright.dev/docs/auth)? 
In case of "yes" I would be glad about how to this :)

**Comment by digracesion** [comment]
I don't think this is specifically related to playwright-lighthouse, but playwright itself.
But to answer your specific question, I found this [video demo](https://www.youtube.com/watch?v=lCb9JoZFpHI) to be really helpful in visualizing how to use storage state
